src/components/interface/dialog/DialogInput.jsx

```diff
@@ -9,6 +9,7 @@ const DialogInput = ({
   disabled,
   fullWidth,
   helperText,
+  InputProps,
```

Comment:
Is there a reason `InputProps` is not proper camel case?

Reply:
That's the name of the props